subprojects/docs/src/docs/stylesheets/dslHtml.xsl

```diff
@@ -36,6 +36,7 @@
 
     <!-- customize the stylesheets to add to the <head> element -->
     <xsl:template name="output.html.stylesheets">
+        <link href="https://fonts.googleapis.com/css?family=Lato:400,700" rel="stylesheet"/>
```

Comment:
Does this also work for offline documentation?

Reply:
BTW, China government blocks `googleapi` access...

Reply:
@blindpirate I'd like to understand how other Gradle sites (or other sites using Google Fonts) behave for you. We use Google Fonts for the Blog and assets.gradle.com for other sites. 

Does the Gradle Blog load for you at all? If it does, do the requests to Google Fonts fail quickly, slowly, or succeed? Do our sites load slowly for you? How does this change when using versus not using the VPN?